com.unity.render-pipelines.high-definition/Runtime/Lighting/VolumetricLighting/VolumetricClouds.compute

```diff
@@ -9,6 +9,7 @@
 #pragma multi_compile _ PHYSICALLY_BASED_SUN
 #pragma multi_compile _ PLANAR_REFLECTION_CAMERA
 #pragma multi_compile _ LOCAL_VOLUMETRIC_CLOUDS
+#pragma multi_compile _ ENABLE_MSAA
```

Comment:
Why not simply have USE_INTERMEDIATE_BUFFER as the multi compile and add that extra intermediate define?